### PR TITLE
Create new VedAstro integration

### PR DESCRIPTION
**Pull request by sengiv**
# Description

Add new integration for getting prediction documents from simple birth time input as string

Fixes # (issue)

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [x] Yes
- [ ] No

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
+"""Simple Web scraper."""
+from typing import List, Optional, Dict, Callable
+
+import requests
+import json
+from llama_index.core.bridge.pydantic import PrivateAttr
+from llama_index.core.readers.base import BasePydanticReader
+from llama_index.core.schema import Document
+from vedastro import *
```
Comment (logan-markewich) [inline llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py:9]: I would prefer specific imports here

File: llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+			predict_node = Document(
+			    text=prediction["Description"],
+			    metadata=predict_meta,
+			    metadata_seperator="::",
+			    metadata_template="{key}=>{value}",
+			    text_template="Metadata: {metadata_str}\n-----\nContent: {content}",
+			)
+
+			# # this is shows difference for understanding output of Documents
```
Comment (logan-markewich) [inline llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py:184]: Can remove these comments

File: llama-index-integrations/readers/llama-index-readers-vedastro/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Simple Website Loader
```
Comment (logan-markewich) [inline llama-index-integrations/readers/llama-index-readers-vedastro/README.md:1]: Is this the correct header?

**Review by logan-markewich — COMMENTED** [review]
Seems like you'll need to run `make lint` to fix linting errors

File: llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py (hunk 1 above)
Comment (sengiv) [reply llama-index-integrations/readers/llama-index-readers-vedastro/llama_index/readers/vedastro/base.py:9]: noted ✅

**Pull request closed without merging**